src/assets/stylesheets/index.scss

```diff
@@ -122,7 +122,7 @@ html {
 }
 
 body {
-  font-family: "Source Sans Pro", $font-family-sans-serif;
```

Comment:
* [ ] Should we also use the simple value in `_variables.scss`, to prevent having two different `sans-serif` families?

Reply:
Alex G. and I settled on the simple `font-family: "Source Sans Pro", sans-serif;` version, which does not need an extra dependency.

Reply:
To clarify: There's still one use of $font-fmily-sans-serif, which we could *get rid of*. In _varaibles.scss.

Reply:
`_varaibles.scss` is part of `bootstrap`. I don't want to modify bootstrap parts.